src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/NestedFieldQueryIT.java

```diff
@@ -170,6 +172,107 @@ public void singleCondition() throws IOException {
         );
     }
 
+    @Test
+    public void regularAndDeepNestedFieldInSelect() throws IOException {
+        assertThat(
+            execute("select title, a.city " +
+                    "from elasticsearch-sql_test_index_employee_nested e, e.projects p, p.address a"),
+            hits(
+                hit(
+                    textMatch("title", null),
+                    innerHits("projects.address",
+                        hit(
+                            textMatch("city", "Seattle")
+                        ),
+                        hit(
+                            textMatch("city", "Boston")
+                        ),
+                        hit(
+                            textMatch("city", "Chicago")
+                        )
+                    )
+                ),
+                hit(
+                    textMatch("title", "Software Eng 2"),
+                    innerHits("projects.address",
+                        hit(
+                            textMatch("city", "Dallas")
+                        )
+                    )
+                )
+            )
+        );
+    }
+
+    @Test
+    public void deepSelectAll() throws IOException {
+        assertThat(
+            execute("SELECT * FROM elasticsearch-sql_test_index_employee_nested e, e.projects p, p.address a"),
```

Comment:
np: I still think using the static constants for index names is better.

Reply:
I think that the most of the problem is that our generated index names are too long.

static constants - why just for index names, why not for fields? statements? Current way is cleanest possible, and doesn't hide anything from reader